datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateEntryGroup.java

```diff
@@ -69,4 +69,3 @@ public static void createEntryGroup(String projectId, String entryGroupId) {
     }
   }
 }
-// [END datacatalog_create_entry_group_tag]
```

Comment:
Seems like this line was removed by mistake.

Reply:
yea made a mistake after running the java formatter